vars/mavenStaticCodeChecks.groovy

```diff
@@ -0,0 +1,50 @@
+import com.sap.piper.PiperGoUtils
+import com.sap.piper.Utils
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String METADATA_FILE = 'metadata/mavenStaticCodeChecks.yaml'
+@Field String STEP_NAME = getClass().getName()
```

Comment:
with this step name we deviate from the typical pattern `<tool><verb><substantive>`

I guess it is not too late to get this also fixed for the `golang` part

Reply:
done